docs/systems.md

```diff
@@ -95,6 +95,12 @@ MareNostrum 5 is the EuroHPC JU supercomputer hosted by the [Barcelona Supercomp
 
 * Aristotle: [General documentation](https://hpc.it.auth.gr/nodes-summary_en/) | [EESSI @ Aristotle](https://hpc.it.auth.gr/software/eessi/)
 
+### Hungary
+
+#### Digital Government Development and Project Management Ltd. (DKF Kft)
+
+* Komondor: [General documentation](https://docs.hpc.dkf.hu/)
```

Comment:
@Atisom Can/should we also link to https://docs.hpc.dkf.hu/software/eessi.html ?

If there's an EESSI-specific page is available for the system, we prefer to link to it too.